Kinder/Registration.xaml.cs

```diff
@@ -220,5 +223,15 @@ private void ShowPassword_Click(object sender, RoutedEventArgs e)
                 password.Password = passwordShow.Text;
             }
         }
+
+        private void GetUsernameHint(string triedUsername)
+        {
+            string possibleUsername1 = triedUsername + 0;
```

Comment:
why is there an addition to zero ?

Reply:
It works as a just a number that adds to username as a string, for example admin if "admin" is taken, it will hint you a new username "admin0", if it will be taken, then another available will be username "admin1" and etc.